common/rakelib/lint.rake

```diff
@@ -17,39 +17,39 @@ namespace :lint do
     tmppath = '/tmp/SwiftLint.pkg'
 
     Utils.run([
-      "curl -Lo #{tmppath} #{url}",
-      "sudo installer -pkg #{tmppath} -target /"
-    ], task)
+                "curl -Lo #{tmppath} #{url}",
+                "sudo installer -pkg #{tmppath} -target /"
+              ], task)
   end
-  
+
   if File.directory?('Sources')
     desc 'Lint the code'
-    task :code => :install do |task|
+    task code: :install do |task|
```

Comment:
Don't change this. Even if technically correct when considering this as a Ruby hash (and only in recent versions of Ruby), `Rake` used that syntax on purpose

Technically, Rake defines a `def task(params)` method internally to define rake tasks, and allows you to write:

* `task :foo do` to define a simple task without dependency. Which in practice is just calling `task()` with a single parameter `:foo` (of type symbol), defining the name of the task
* `task :code => :install do` to define a task with a single dependency. The use of `=>` is expected to give the visual meaning of dependency. In practice what you do is pass a single parameter to that `task` method which happens to be a Hash (dictionary) with a single key (`:code`)  and a value (`:install`)… but that's implementation detail
* `task :code => [:x, :y, :z] do` to define multiple dependencies, once again that `=>` arrow is supposed to represent the visual cue of dependency, even if in practice what you do is pass a single parameter of type `Hash` with a single key `:code`, whose value is an array of 3 values… but again that's an implementation detail

So in practice Rake chose to use that implementation detail _because_ that `=>` was a nice visual cue for dependency (and not because it was technically a hash, which I actually never realised it was in that case until recently). So I think for the special case of `task` declarations, it's better to keep the `=>` syntax, whatever rubocop says.